src/test/index.test.ts

```diff
@@ -3,21 +3,52 @@ import { expect } from 'chai';
 import chai = require('chai');
```

Comment:
nao da pra fazer por import?

```suggestion
import { expect } from 'chai';
```